vuejs/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "parcel-vuejs-example",
+  "version": "1.0.0",
+  "description": "Parcel Bundler Example with Vue.js",
+  "main": "src/main.js",
+  "scripts": {
```

Comment:
`start` and `build` won't work on windows. Could be fixed by using `cross-env`

Reply:
Okay, I fix it.

Reply:
Also, we're waiting for this issue https://github.com/lc60005457/parcel-plugin-vue/issues/5 for create more advanced Vue.js example (with images, who bugged now if you add in components).